### PR TITLE
update home page, update info site, remove website search option and …

### PR DESCRIPTION
**Pull request by DanielPalafox**
https://trello.com/c/OIlbX1qu/6829-registry-changes-needed-for-info-site-migration-5

### CODE REVIEW

**Comment by DanielPalafox** [comment]
[Registry changes needed for info site migration (5)](https://trello.com/c/OIlbX1qu/6829-registry-changes-needed-for-info-site-migration-5)

File: src/app/layout/header/header.component.html [hunk 1]
```diff
@@ -70,6 +70,7 @@
                 active: item.activeRoute === currentRoute
               }"
               mat-button
+              class="no-hover-effect"
```
[the remark this answers is not in the document]
Comment (leomendoza123) [reply src/app/layout/header/header.component.html:73]: this is also affecting mobile

File: src/app/layout/search/search.component.scss [hunk 2]
```diff
@@ -86,13 +96,18 @@
   :host {
     .search-box {
       padding: 12px 0px;
+      justify-content: center;
+      max-width: -webkit-calc(100% - 28px);
+      max-width: -moz-calc(100% - 28px);
```
Comment (leomendoza123) [inline src/app/layout/search/search.component.scss:101]: Seems like vendor prefix is not required since the cal is not an experimental feature anymore

**Comment by leomendoza123** [comment]
@DanielPalafox One more thing, since this is under a togglz we need to create a new translations string apart from `ngOrcid.search` with a togglz instead of overwriting the old one

File: src/app/layout/header/header.component.html [hunk 3]
```diff
@@ -138,6 +139,7 @@
         </li>
       </ul>
     </div>
+    <mat-divider class="divider"></mat-divider>
```
[the remark this answers is not in the document]
Comment (DanielPalafox) [reply src/app/layout/header/header.component.html:142]: Ok

File: src/app/layout/header/header.component.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (DanielPalafox) [reply src/app/layout/header/header.component.html:73]: That class removes the default hover effect but I put other effects as was specified in the design
image
image

[the remark this answers is not in the document]
Comment (leomendoza123) [reply src/app/layout/header/header.component.html:73]: Sorry, Daniel, the issue is with focus, not the hover ( I always confuse those names) 
Without a focus effect, the keyboard users won't be able to know where they are currently focusing.

File: src/app/layout/search/search.component.scss (hunk 2 above)
Comment (DanielPalafox) [reply src/app/layout/search/search.component.scss:101]: Ok, ok

File: src/app/layout/footer/footer.component.html [hunk 4]
```diff
@@ -50,15 +50,22 @@
         [ngClass]="{ 'space-around': !platform.columns12 }"
       >
         <a
-          class="col s4 l-auto m8  "
+          class="col s4 l-auto m8"
```
[the remark this answers is not in the document]
Comment (DanielPalafox) [reply src/app/layout/footer/footer.component.html:53]: Ok, I will use the `A11yLinkModule`

File: src/app/layout/header/header.component.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (DanielPalafox) [reply src/app/layout/header/header.component.html:73]: Ok, I will fix it.

**Pull request merged**